test/JDBC/upgrade/14_3/preparation/bbf_view_def_upgrade-vu-prepare.sql

```diff
@@ -0,0 +1,10 @@
+CREATE VIEW view_3135_upd AS SELECT 1;
```

Comment:
It seems all prepare files are same. If that is the case, add single file to input directory and add corresponding entry to schedule file

Reply:
But output are different. If I'll add it to schedule file then it will compare it's output with common expected file (i.e. bbf_view_def_upgrade-vu-prepare.out) and fail.